File: formMain.vb
Public Class formMain

    Private Sub RestoreDatabase_Click(sender As Object, e As EventArgs) Handles buttonRestoreDatabase.Click
        Me.Cursor = Cursors.WaitCursor

        formRestoreDatabase.Show()
        Me.Close()

        Me.Cursor = Cursors.Default
    End Sub
End Class

File: FormMain.cs
namespace CSSqlDbBackupTools
{
    public partial class FormMain : Form
    {
        public FormMain()
        {
            InitializeComponent();
        }

        private void ButtonRestoreDatabase_Click(object sender, EventArgs e)
        {
            this.Cursor = Cursors.WaitCursor;
            FormRestoreDatabase formRestoreDatabase = new();
            formRestoreDatabase.ShowDialog();
            this.Cursor = Cursor.Current;
        }
    }
}
